spec/features/session_spec.rb

```diff
@@ -314,6 +314,19 @@
       end
     end
 
+    describe "Node#obscured?" do
```

Comment:
I suggest to add tests for a node that is not obscured, for a node that contains children and is not obscured by them, and for a node in different frame

Reply:
I think those scenarios are all covered by the upstream specs in [`capybara/lib/capybara/spec/session
/node_spec.rb`](https://github.com/teamcapybara/capybara/blob/0480f90168a40780d1398c75031a255c1819dce8/lib/capybara/spec/session/node_spec.rb#L312-L384) (I updated `spec_helper.rb` to remove the `node #obscured?` filter). I will double-check though!

Reply:
@toy there were a couple upstream specs that I had not enabled - now fixed.

> for a node that contains children and is not obscured by them

I also enabled a test for the above - that was disabled on CI in the upstream suite.